detect_secrets/plugins/common/initialize.py

```diff
@@ -16,6 +11,7 @@
 from ..stripe import StripeDetector                         # noqa: F401
 from detect_secrets.core.log import log
 from detect_secrets.core.usage import PluginOptions
+from detect_secrets.plugins.common.util import get_mapping_from_secret_type_to_class_name
```

Comment:
Nit: Since we use relative imports above, for within the plugins folder, so I think we can do `from .util` in this case

Reply:
Changed as suggested